components/Select/Trigger.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+function getTriggerClass({ color }) {
+  return classNames({
+    truncate: true,
+    'btn--fill': true,
+    'btn-dropdown-gray':  color === 'gray',
+    'btn-dropdown-white': color === 'white',
+    'btn-dropdown-teal':  color === 'teal',
+    'c-select--trigger': true,
+  });
+}
+
+const Trigger = ({ color, isOpen, onOpenToggle, triggerLabel, triggerPlaceholder }) => (
+  <span className={getTriggerClass({ color })} onClick={() => onOpenToggle(!isOpen)}>{triggerLabel || triggerPlaceholder}</span>
+);
+
+Trigger.propTypes = {
+  color: PropTypes.oneOf([ 'gray', 'white', 'teal' ]).isRequired,
+  isOpen: PropTypes.bool.isRequired, // TODO: could remove this propType, but it would probably be useful to anyone specifying a custom <Trigger> element
```

Comment:
I think I'll remove this `todo:` comment and keep the propType there. It's passed into the custom trigger. If you were making a custom trigger, you would likely want it to be able to know if the select `isOpen`